### PR TITLE
[TRACE CONTROL] Add JSONRPC API

### PR DESCRIPTION
**Pull request by Damian-Danylko**

### CODE REVIEW

File: TraceControl/TraceControl.h [hunk 1]
```diff
@@ -6,7 +6,12 @@ namespace WPEFramework {
 
 namespace Plugin {
 
-    class TraceControl : public PluginHost::IPlugin, public PluginHost::IWeb {
+namespace {
+    constexpr auto* kStatusMethodName = _T("status");
```
Comment (pwielders) [inline TraceControl/TraceControl.h]: Can we for these very simple types just use the actual type, TCHAR in this case. Also as we do not expect this value to be a nullptr, the most verbose definition would be:
constexpr TCHAR kStatusMethodName[] = _T("status");

File: TraceControl/TraceControl.h [hunk 2]
```diff
@@ -810,6 +815,25 @@ namespace Plugin {
         };
         class Data : public Core::JSON::Container {
         public:
+            class StatusParam final : public Core::JSON::Container {
+            public:
+                StatusParam()
+                    : Core::JSON::Container()
+                    , Module("")
```
Comment (pwielders) [inline TraceControl/TraceControl.h]: If it is empty string, just use the default constructor ("") is not needed.

File: TraceControl/TraceControl.h (hunk 1 above)
Comment (Damian-Danylko) [reply TraceControl/TraceControl.h]: Done

File: TraceControl/TraceControl.h (hunk 2 above)
Comment (Damian-Danylko) [reply TraceControl/TraceControl.h]: Done

**Pull request merged**